### PR TITLE
Restore `one-click-review-submissions`

### LINKED ISSUE #6618: `one-click-review-submission` feature broken
### Description

Oops

Screenshot 4

```
TypeError: Cannot read properties of undefined (reading 'append')
    at addButtons (chrome-extension://jfnbjdajagmjideggedlljgaddlmffhd/refined-github.js:10503:15)
    at init (chrome-extension://jfnbjdajagmjideggedlljgaddlmffhd/refined-github.js:10568:3)
    at runFeature (chrome-extension://jfnbjdajagmjideggedlljgaddlmffhd/refined-github.js:3793:22)
    at setupPageLoad (chrome-extension://jfnbjdajagmjideggedlljgaddlmffhd/refined-github.js:3805:11)
    at chrome-extension://jfnbjdajagmjideggedlljgaddlmffhd/refined-github.js:3857:15
```

Probably it's what was being requested in https://github.com/refined-github/refined-github/issues/6048#issuecomment-1536531740

### How to replicate the issue + URL

https://github.com/refined-github/refined-github/pull/6617/files

### Extension version

23.4.26

### Browser(s) used

Chrome, Safari

### PR DESCRIPTION
**Pull request by fregante**
- Fixes #6618

## Test URLs

- https://github.com/refined-github/refined-github/pull/6619/files

## Screenshot

Screenshot 3
Screenshot 2

### CODE REVIEW

**Review by fregante — COMMENTED** [review]
Updated screenshot

Screenshot 7

File: source/features/one-click-review-submission.tsx [hunk 1]
```diff
@@ -9,11 +9,13 @@ import looseParseInt from '../helpers/loose-parse-int.js';
 
 function addButtons(radios: HTMLInputElement[]): void {
 	const form = radios[0].form!;
-	const container = select('.form-actions', form)!;
+	const container
+		= form.closest('.SelectMenu')?.querySelector('.form-actions')
+		?? select('.form-actions', form)!; // TODO: Drop after September 2023
```
Comment (fregante) [inline source/features/one-click-review-submission.tsx:14]: The action buttons are no longer inside the `form`, they're now linked via `form` attribute

File: source/features/one-click-review-submission.tsx [hunk 2]
```diff
@@ -22,11 +24,6 @@ function addButtons(radios: HTMLInputElement[]): void {
 		);
 	}
 
-	// "Comment" button must be first
-	if (radios.length > 1) {
-		radios.push(radios.shift()!);
-	}
```
Comment (fregante) [inline source/features/one-click-review-submission.tsx:28]: Very long in the tooth. Long story:

- https://github.com/refined-github/refined-github/pull/5054#issuecomment-963575566

**Comment by take-five** [comment]
Hi @fregante 
I think the order of actions has changed. It used to be that "Approve" button was the right-most, but now the right-most button is "Comment".
Was this change of order intentional? It's kind of a major change in the UI :-)

**Comment by fregante** [comment]
> Was this change of order intentional? It's kind of a major change in the UI :-)

Thanks for the comment, yes this was intentional and it was supposed to happen a few years ago (see the [related issue](https://github.com/refined-github/refined-github/pull/5054#issuecomment-963575566) above)

While the change is expected to be annoying, this finally restores the expected order and matches GitHub’s own UI. Refer to the [comparison screenshot](https://user-images.githubusercontent.com/1402241/236627732-df341ff7-cd98-4cd0-a579-722d1fffa5cf.png) above.

**Comment by fregante** [comment]
Will not be changed back, the pain is temporary 😅

**Comment by take-five** [comment]
Is it at least possible to revert the coloring so that the "Approve" button is the one that is green and emphasized? When I click on a big green blob, I subconsciously expect that the PR will be approved (consistent with other buttons that change the state of PRs - for example, "Merge" button is also a green blob).

**Comment by fregante** [comment]
The coloring has been unchanged for years. This PR only changed the position of the comment button. This is the reason why we have the current colors:

- https://github.com/refined-github/refined-github/issues/5010

**Comment by take-five** [comment]
You're right. But it's still super confusing.

**Comment by omBratteng** [comment]
@mfn @take-five  since the extension allows for custom CSS, it can be "fixed" with some simple CSS. I had to do it because I do not well with changes, my whole body irks and I need to get it back to the way it was.

So you can use this piece of CSS to make it look like before
```css
#review-changes-modal .form-actions {
    display: flex;
    justify-content: flex-end;
}

#review-changes-modal .form-actions .review-cancel-button {
    order: -1;
}

#review-changes-modal .form-actions .review-title-with-count {
    order: 0;
    margin-right: auto;
}

#review-changes-modal .form-actions button[value="comment"] {
    order: 1;
    margin-right: 5px;
}

#review-changes-modal .form-actions button[value="reject"] {
    order: 2;
    margin-right: 5px;
}

#review-changes-modal .form-actions button[value="approve"] {
    order: 3;
    margin-right: 0;
}
```

**Pull request merged**